CMakeLists.txt

```diff
@@ -198,57 +174,49 @@ set(SDLSOUND_ALL_SRCS
 )
 
 option(SDLSOUND_BUILD_STATIC "Build static library" TRUE)
-if(OS2) # no static library on os/2
-    set(SDLSOUND_BUILD_STATIC FALSE)
-endif()
 if(SDLSOUND_BUILD_STATIC)
-    add_library(SDL2_sound-static STATIC ${SDLSOUND_ALL_SRCS})
-    add_library(SDL2_sound::SDL2_sound-static ALIAS SDL2_sound-static)
+    add_library(SDL3_sound-static STATIC ${SDLSOUND_ALL_SRCS})
+    add_library(SDL3_sound::SDL3_sound-static ALIAS SDL3_sound-static)
     # Don't rename this on Windows, since DLLs will also produce an import
-    #  library named "SDL2_sound.lib" which would conflict; Unix tend to like
+    #  library named "SDL3_sound.lib" which would conflict; Unix tend to like
     #  the same library name with a different extension for static libs, but
     #  Windows can just have a separate name.
     if(NOT MSVC)
-        set_target_properties(SDL2_sound-static PROPERTIES OUTPUT_NAME "SDL2_sound")
+        set_target_properties(SDL3_sound-static PROPERTIES OUTPUT_NAME "SDL3_sound")
     endif()
-    target_include_directories(SDL2_sound-static PUBLIC
+    target_include_directories(SDL3_sound-static PUBLIC
         $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
-        $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/SDL2>
         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src>)
-    target_include_directories(SDL2_sound-static PUBLIC "$<BUILD_INTERFACE:${SDL2_INCLUDE_DIRS}>")
-    target_link_libraries(SDL2_sound-static PRIVATE "$<BUILD_INTERFACE:${SDL2_LIBRARIES}>" ${OPTIONAL_LIBRARY_LIBS} ${OTHER_LDFLAGS})
-    set(SDLSOUND_LIB_TARGET SDL2_sound-static)
+    target_include_directories(SDL3_sound-static PUBLIC "$<BUILD_INTERFACE:${SDL3_INCLUDE_DIRS}>")
+    target_link_libraries(SDL3_sound-static PRIVATE SDL3::SDL3 ${OPTIONAL_LIBRARY_LIBS} ${OTHER_LDFLAGS})
+    set(SDLSOUND_LIB_TARGET SDL3_sound-static)
 endif()
 
 option(SDLSOUND_BUILD_SHARED "Build shared library" TRUE)
 if(SDLSOUND_BUILD_SHARED)
-    add_library(SDL2_sound SHARED ${SDLSOUND_ALL_SRCS} ${SDLSOUND_SHARED_SRCS})
-    add_library(SDL2_sound::SDL2_sound ALIAS SDL2_sound)
-    set_target_properties(SDL2_sound PROPERTIES MACOSX_RPATH 1)
-    set_target_properties(SDL2_sound PROPERTIES VERSION ${SDLSOUND_VERSION})
-    set_target_properties(SDL2_sound PROPERTIES SOVERSION ${SDLSOUND_SOVERSION})
+    add_library(SDL3_sound SHARED ${SDLSOUND_ALL_SRCS} ${SDLSOUND_SHARED_SRCS})
```

Comment:
SDL3 and all SDL3 satellite libraries create a `SDL_xxx::SDL_xxx-shared` and `SDL_xxx::SDL_xxx-static` target,
with `SDL_xxx::SDL_xxx` an alias to one of these (depending on what is available).

Should SDL_sound do the same?

Reply:
Really don't know. I'm not much of a cmake guy as you know :)

Reply:
No problem, you do a lot with Makefiles ;)